### ISSUE TITLE
Maps not working

### ISSUE THREAD

**Issue by alistaireverett** [metno/icewatch#3]
Alaskamapped wms server no longer available. Need to transition to a new wms server.

L31- 34 in app/assets/javascripts/cruises.js.coffee:

```
 L.tileLayer.wms('http://wms.alaskamapped.org/gina/bdl?GOGC=22F4942A5CC6',`
   layers: 'BestDataAvailableLayer'
   format: 'image/jpeg'
   continuousWorld: true).addTo(map)
```

**Comment by alistaireverett** [comment]
Fixed in 6021d09 using EUMETSAT wms server instead.

**Comment by alistaireverett** [comment]
Fixed in [new_wms](https://github.com/metno/icewatch/tree/new_wms) branch.

**Comment by alistaireverett** [comment]
merged into master #13
